src/appRegistrationEffects.js

```diff
@@ -1,28 +1,26 @@
-import { createEffect, createSignal } from "solid-js";
+import { createEffect, createSignal } from 'solid-js';
 
-import { app_id, server_url, token1 } from "./storageSignals";
-import { send } from "./stateSignal";
+import { app_id, server_url, token1 } from './storageSignals';
+import { send } from './stateSignal';
 
-export const [oauthUrl, setOauthUrl] = createSignal("");
+export const [oauthUrl, setOauthUrl] = createSignal('');
 
 createEffect(() => {
-  const is_production = window.location.hostname === "api.deriv.com";
-  const production_app_id = 31063;
-  if (is_production) {
-    localStorage.setItem("app_id", production_app_id);
-    localStorage.setItem("server_url", "https://green.binaryws.com");
-  }
-  const loginUrl = () => {
-    if (is_production) {
-      return `https://oauth.deriv.com/oauth2/authorize?app_id=${production_app_id}&l=EN&brand=deriv`;
+    const is_production = window.location.hostname === 'api.deriv.com';
+    const production_app_id = 31063;
+    localStorage.setItem('app_id', production_app_id);
+    localStorage.setItem('server_url', 'https://green.binaryws.com');
```

Comment:
we discussed that in this file setItem will trigger only on registration page, and we wanted default values to be set in storageSignals.js that runs on first page load, anything changed?